sci-misc/boinc/files/boinc.init

```diff
@@ -11,10 +11,10 @@ depend() {
 
 
 create_work_directory() {
-	if [[ ! -d "${RUNTIMEDIR}" ]]; then
+	if [ ! -d "${RUNTIMEDIR}" ]; then
```

Comment:
Hmm... I don't know. RUNTIMEDIR is a fixed path where BOINC stores all its data. Quite contrary to tmp I daresay.
However, a tmpfiles.d config line like
````
d ${RUNTIMEDIR} 0755 boinc boinc - -
````
would not get cleaned. However, the RUNTIMEDIR must be configurable, and thus changeable. Actually I already did use this for some tests of my various accounts, having switched RUNTIMEDIR a few times.

What is 'checkpath'? I can't seem to find it.